server/php-worker.cpp

```diff
@@ -326,6 +333,7 @@ void php_worker_run(php_worker *worker) {
       }
       case run_state_t::finished: {
         vkprintf (2, "php script [req_id = %016llx]: OK (still can return RPC_ERROR)\n", worker->req_id);
+        php_script_run_shutdown_functions();
         script_result *res = php_script_get_res(php_script);
```

Comment:
Now it differs from old behavior in case when shutdown functions append something to response
For example:
```php
<?php

/**
 * @kphp-required
 */
function at_finish()
{
    echo " FINISHED\n";
}
register_shutdown_function('at_finish');

echo "Answer";
```
In vanilla php and old kphp HTTP response contains 'Answer FINISHED'
Now it contains 'Answer' only

To be honest, I don't know how much it's important, but anyway